set-web/src/test/java/eu/europeana/set/web/service/impl/UserSetServiceImplTest.java

```diff
@@ -112,18 +113,19 @@ public void testPaginationForPage2() throws Exception {
 	assertTrue(lastPage == 4);
 
 	userSetQuery.setPageNr(2);
-
-	String first = userSetService.buildPageUrl(REQUEST_URL , 0, userSetQuery.getPageSize());
-	String last = userSetService.buildPageUrl(REQUEST_URL, lastPage, userSetQuery.getPageSize());
+	LdProfiles profile = LdProfiles.MINIMAL;
+	// check with minimal profile
+	String first = userSetService.buildPageUrl(REQUEST_URL , 0, userSetQuery.getPageSize(), profile);
+	String last = userSetService.buildPageUrl(REQUEST_URL, lastPage, userSetQuery.getPageSize(), profile);
 	String next = userSetService.buildPageUrl(REQUEST_URL, userSetQuery.getPageNr() + 1,
-		userSetQuery.getPageSize());
+		userSetQuery.getPageSize(), profile);
 	String prev = userSetService.buildPageUrl(REQUEST_URL, userSetQuery.getPageNr() - 1,
-		userSetQuery.getPageSize());
+		userSetQuery.getPageSize(), profile);
 	String curr = userSetService.buildPageUrl(REQUEST_URL, userSetQuery.getPageNr(),
-		userSetQuery.getPageSize());
+		userSetQuery.getPageSize(), profile);
 
 	BaseUserSetResultPage<?> result = userSetService.buildResultsPage(userSetQuery, resultSet, REQUEST_URL, "",
-			Collections.singletonList(LdProfiles.MINIMAL), authentication);
+			Collections.singletonList(profile), authentication);
 
 	assertTrue(StringUtils.equals(first, result.getPartOf().getFirst()));
 	assertTrue(StringUtils.equals(last, result.getPartOf().getLast()));
```

Comment:
can we add a check for the partOf? it should not contain profile parameter

Reply:
done